roles/StackStorm.st2/tasks/main.yml

```diff
@@ -100,3 +100,15 @@
     state: started
   loop: "{{ st2_services }}"
   tags: st2
+
+# https://github.com/ansible/ansible/issues/41313#issuecomment-520891625
+- name: Flush handlers to prepare to run st2 there are packs to install
+  include_tasks: flush_handlers.yml
+  when: st2_packs is defined
+
+- name: Install ST2 add-on Packs
+  import_tasks: packs.yml
+  when: st2_packs is defined and
+        ((ansible_distribution == "RedHat" and ansible_distribution_major_version == '7') or
```

Comment:
Including particular OSes in this check is error-prone as other OS versions could be added.
Please check if target OS is EL6 explicitly instead as that's only one that had issues:

See an existing code example: https://github.com/StackStorm/ansible-st2/blob/684d0c3052b8fc3503baec06c9eaed3fd32e9d6b/roles/StackStorm.st2/tasks/main.yml#L11

Reply:
ok refactored to `exclude` CentOS6